protoc-gen-go/generator/generator.go

```diff
@@ -92,6 +92,12 @@ func RegisterPlugin(p Plugin) {
 	plugins = append(plugins, p)
 }
 
+// An ImportPath is the import path of a Go package. e.g., "github.com/golang/protobuf".
+type ImportPath string
```

Comment:
Rename as `GoImportPath`? Clearly distinguishes it from a proto import path, which is a legitimate concept.

Reply:
done